pandas/tests/groupby/test_groupby.py

```diff
@@ -2541,7 +2542,10 @@ def test_groupby_column_index_name_lost(func):
     "infer_string",
     [
         False,
-        True,
+        pytest.param(
+            True,
+            marks=pytest.mark.skipif(not pa_installed, reason="arrow not installed"),
```

Comment:
I just realized we have a `td.skip_if_no` mark as well

Reply:
does this work inside of pytest.param as well?

Reply:
It should, yeah. It returns a `skipif` marker

Reply:
Yep, thx